src/utils/db.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+require('dotenv').config();
+
+const { Sequelize } = require('sequelize');
+
+const dbHost = process.env.DB_HOST || 'localhost';
+const dbName = process.env.DB_NAME || 'postgres';
+const dbUser = process.env.DB_USER || 'postgres';
+const dbPassword = process.env.DB_PASSWORD || 'pass';
+
+const sequelize = new Sequelize(dbName, dbUser, dbPassword, {
+  host: dbHost,
+  dialect: dbName,
```

Comment:
In the dialect field, you should specify the type of database you are using (e.g. 'postgres', 'mysql', 'sqlite', etc.) instead of using the dbName variable. 🛠

consider renaming to variable dbDialect or dbEngine